tests/integration/ssh/test_grains.py

```diff
@@ -21,5 +21,8 @@ def test_grains_items(self):
         test grains.items with salt-ssh
         '''
         ret = self.run_function('grains.items')
-        self.assertEqual(ret['kernel'], 'Linux')
+        grain = 'Linux'
+        if salt.utils.platform.is_darwin():
```

Comment:
Ah, 2017.7 still has `is_darwin` in the `salt.utils` init file. So `platform` here won't work on this branch.

Reply:
lol thats what i get for only testing on oxygen. Will fix. Do i need to do another PR for oxygen or will that be handled in the merge forward?